infra/utils/versioning.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (pkgName) => {
+  const pkgJson = require(`../../packages/${pkgName}/package.json`);
+  const details = ['workbox', pkgName, pkgJson.version];
+  return `function v(){v._version='${details.join(':')}'};v();`;
```

Comment:
Quick sanity check: are there any concerns about the symbol `v` ending up clashing with anything else in the same scope? Does `v` get added in before or after the uglification takes place/